### DISCUSSION TITLE
Stakeholder Mastering

### DISCUSSION THREAD

**Discussion by jtylwalk** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#131] [Suggestions] [answered]
Relates to https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/discussions/110

If multiple cap table providers are feeding data to a central downstream system, how could one ensure that the single stakeholder is consistently represented across multiple feeds?

A hypothetical example:

  - employee works at a company and is granted options in company A, where the provider is X
  - employee moves to another company and is granted options in company B, where the provider is Y

Then the goal would be to unify the data from provider X and Y, ideally bringing together all the data for the single employee.

The spec describes the `issuer_assigned_id` - but it could be the case that an individual was issued different identifiers for each company. Hypothetically the tax id might be able to be used to bridge the feed data, if it is consistently configured for the employee with both providers.

The broader question: is there any thought around a common identification mechanism could be introduced to resolve the need for 'matching' of stakeholders across feeds from different providers? I understand it would possibly be complicated by the fact that there isn't a central authority for identification in this scenario.

**Comment by JSv4** [comment, marked as answer]
@jtylwalk, another great idea that would require some coordination in the industry or someone to step up and issue these kinds of identifiers. I think that's beyond what we're trying to do, but I could see some amazing benefits for analytics and data portability if there were a central repository for identifiers like this.
